en/streaming-integrator/docs/examples/handling-requests-with-errors.md

```diff
@@ -148,6 +148,32 @@ To create and deploy a Siddhi application, follow the steps below:
        select count() as totalCount
        insert into TotalCountStream;
     ```
+    
+    ```
+        @App:name("StoreRdbmsError")
+        @App:description('Receive events via simulator and persist the received data in the store.')
```

Comment:
Let's say `Receive events via HTTP source and persist ...`

Reply:
Fixed in a5ef3f0